src/pages/FirstPage.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from 'react';
+import { StepContext } from '../App';
+import styled from 'styled-components';
+import theme from '../styles/theme';
+
+const FirstPage = () => {
+  const { step, setStep } = useContext(StepContext);
+
+  return (
+    <Container>
+      <CardContainer>
+        <SubTitle>어르신 돌보미</SubTitle>
+        <Title>케어코디</Title>
+      </CardContainer>
+      <LandingButton onClick={() => setStep(step + 1)}>신청하기</LandingButton>
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  width: 768px;
+  height: 100vh;
+  margin: 0 auto;
+  font-weight: 600;
+`;
+
+const CardContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  font-size: 2rem;
+  line-height: 3rem;
+`;
+
+const SubTitle = styled.div`
+  color: ${theme.fontColor};
```

Comment:
`theme`을 곧바로 불러와서 사용하는게 아니라 prop로 전달된 theme을 이용하셔야 합니다.
```suggestion
  color: ${({ theme }) => theme.fontColor};
```

Reply:
네 수정하였습니다!